Mathlib/Analysis/InnerProductSpace/MeanErgodic.lean

```diff
@@ -0,0 +1,67 @@
+/-
+Copyright (c) 2023 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury Kudryashov
+-/
+import Mathlib.Analysis.InnerProductSpace.Projection
+import Mathlib.Dynamics.BirkhoffSum.NormedSpace
+
+/-!
+# Mean Ergodic Theorem in a Hilbert Space
+
+In this file we prove the von Neumann Mean Ergodic Theorem for an operator in a Hilbert space.
+It says that for a linear isometry `f : E →ₗᵢ[𝕜] E` of a Hilbert space,
```

Comment:
You don't need a linear isometry, it is enough to have a contraction (i.e., an operator with norm at most 1). See for instance Krengel, Ergodic Theorems, Theorem 1.4 on Page 4.

Reply:
Generalized.